nixos/roles/nix-remote-builder.nix

```diff
@@ -26,14 +26,18 @@ in
       { domain = "*"; item = "nofile"; type = "-"; value = "20480"; }
```

Comment:
Does this even have any effect for the nix-daemon service that hosts the builds?